specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/expressRouteCircuit.json

```diff
@@ -2248,6 +2248,10 @@
           "readOnly": true,
           "type": "string",
           "description": "The authorization status of the Circuit."
+        },
+        "enableDirectPortRateLimit": {
```

Comment:
>enableDirectPortRateLimit [](http://example.com/codeflow?start=9&length=25)

Recommend enum over boolean. Consider something like directPortDataPolicy: { None, RateLimited, ... }

Reply:
This contract was already approved in private repo. Further downstream changes were made based on this approval. https://github.com/Azure/azure-rest-api-specs-pr/pull/12420